UnityProject/Assets/Scripts/HealthV2/Living/BodyParts/BodyPart.cs

```diff
@@ -24,14 +24,43 @@ public partial class BodyPart : MonoBehaviour, IBodyPartDropDownOrgans
 		[HideInInspector] private readonly List<BodyPart> containBodyParts = new List<BodyPart>();
 		public List<BodyPart> ContainBodyParts => containBodyParts;
 
+
+
 		/// <summary>
 		/// Storage container for things (usually other body parts) held within this body part
 		/// </summary>
 		[HorizontalLine] [Tooltip("Things (eg other organs) held within this")]
 		public ItemStorage OrganStorage = null;
 
+		[SerializeField, Tooltip(
+			 " Could you splatter sit on the player and Could the acid touch body parts contained in this body part ")]
+		private bool isOpenAir = false;
```

Comment:
...Isn't this the same as surface?

Reply:
well all body parts contained in this will be the surface body parts basically